src/tests/ftest/pool/destroy_tests.py

```diff
@@ -374,6 +374,43 @@ def test_destroy_connected(self):
         self.assertTrue(
             exception_detected, "No exception when deleting a connected pool")
 
+    def test_forcedestroy_connected(self):
+        """Forcibly destroy pool with connected client.
+
+        Test destroying a pool that has a connected client with force == true.
+        Should pass.
+
+        :avocado: tags=all,medium,pr
+        :avocado: tags=pool,destroy,forcedestroyconnected
+        """
+        hostlist_servers = self.hostlist_servers[:1]
+
+        # Start servers
+        self.start_servers({self.server_group: hostlist_servers})
+
+        # Create the pool
+        self.validate_pool_creation(hostlist_servers, self.server_group)
+
+        # Connect to the pool
+        self.assertTrue(
+            self.pool.connect(), "Pool connect failed before destroy")
+
+        # Destroy pool with direct API call (no disconnect)
+        self.log.info("Attempting to forcibly destroy a connected pool")
+        exception_detected = False
+        try:
+            self.pool.pool.destroy(1)
+        except DaosApiError as result:
```

Comment:
(pylint-unused-variable) Unused variable 'result'

Reply:
for unused variables, put a leading underscore (e.g. _result)

Reply:
I will try this. However it is used below in the self.log.info call. Obviously I am missing something perhaps about the scope of variables in python.

Reply:
Actually Jeff and I discussed this and decided to keep the code as is. The variable is used in self.log.info() below. Also in a companion PR 2171 (for the release/0.9 branch) this code was not flagged with the same warning.